flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/pom.xml

```diff
@@ -175,6 +189,12 @@ limitations under the License.
             <version>${testcontainers.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>fastjson</artifactId>
+            <version>1.2.83</version>
```

Comment:
The fastjson 1 library is deprecated and prone to security issues. Please consider using existing Jackson library or latest FastJson 2.